### PR TITLE
Release fixes

### PR DESCRIPTION
**Pull request by noahmanger**
## Summary
- Menu improvements:
  - Content changes
  - Adding typeahead to candidate search
  - Removing mega menu from DOM on window resize
  - Updates link to individual contributions page to include `is_individual` and `max_date` filters
- Fixing broken icons in IE
  - Adds `charset=UTF8` to the SVG strings generated by the Gulp plugin (required for IE)
  - Removes the Gulp step that removed dimensions from SVGs — IE requires these to be set in order scale the icons using `background-size`
- Adjusts spacing on messages in filter panel

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -11,9 +11,6 @@ gulp.task('minify-icons', function() {
     .pipe(svgmin({
       plugins: [
         {
-          removeDimensions: true
```
Comment (adborden) [inline gulpfile.js]: By removing this, do we still need to set the `background-size` on a per-icon basis?

File: js/site-nav.js [hunk 2]
```diff
@@ -65,6 +67,14 @@ SiteNav.prototype.initMegaMenu = function() {
       focusClass: 'is-focus',
       openClass: 'is-open'
     });
+
+    new typeahead.Typeahead('.js-menu-search', 'candidates', '/data/');
+  }
+};
+
+SiteNav.prototype.destroyMegaMenu = function() {
+  if ( $('body').width() < helpers.BREAKPOINTS.LARGE ) {
```
Comment (adborden) [inline js/site-nav.js]: Will the menu come back on it's own if you expand the window back above the breakpoint?

File: js/templates/nav-data.hbs [hunk 3]
```diff
@@ -65,8 +70,8 @@
         </div>
         <ul>
           <li class="mega__item"><a href="{{webAppUrl}}/filings">All filings</a></li>
-          <li class="mega__item is-disabled">Authorized committees</li>
-          <li class="mega__item is-disabled">Unauthorized committees</li>
+          <li class="mega__item is-disabled">Authorized committee reports</li>
+          <li class="mega__item is-disabled">Unauthorized committees reports</li>
```
Comment (adborden) [inline js/templates/nav-data.hbs]: Is the copy intentionally use different pluralization? Authorized singular committee vs Unauthorized plural committees?

Comment (noahmanger) [inline js/templates/nav-data.hbs]: Oops, that was a mistake. Thanks!

File: js/site-nav.js (hunk 2 above)
Comment (noahmanger) [inline js/site-nav.js]: Nope. The library we're using doesn't have a `destroy()` method, so bringing it back re-initailizes it all together and rebinds all the event handlers and everything. I think it's ok though. It's more of a temporary fix until we get the mobile version going.

File: gulpfile.js (hunk 1 above)
Comment (noahmanger) [inline gulpfile.js]: Not positive, but I think it might. I want to wait until after launch to mess with it though.

**Pull request merged**